api/working-with-extensions/publishing-extension.md

```diff
@@ -212,14 +212,25 @@ Verifying your publisher will:
 
 ![Verified publisher indicators in VS Code](images/publishing-extension/verified-publishers.png)
 
+To verify a publisher:
+
+1. Visit the Visual Studio Marketplace publisher [management page](https://marketplace.visualstudio.com/manage).
+2. Select or create a publisher you wish to verify.
+3. Input an [eligible domain](#eligible-domains) in the **Verified domain** field and save your changes.
+4. Select **Verify**.
```

Comment:
"Select Verify" maybe merge with the previous step.
This is a simple process, so making it look like it has less steps maybe helps?

Reply:
done 👍